.github/workflows/cargo.yml

```diff
@@ -0,0 +1,29 @@
+name: cargo
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "Modules/cpython-sys/**"
```

Comment:
```suggestion
      - "Modules/cpython-sys/**"
      - "Modules/_base64/**"
```
We should probably also include the base64 acceleration module.

Reply:
Does it make more sense to do `**/*.rs` + `**/Cargo.toml` or other files?

Needing to list every rust-using module here seems like an easy way to get out of date?